lib/utils.pm

```diff
@@ -4,11 +4,12 @@ use base Exporter;
 use Exporter;
 use strict;
 use testapi;
+use File::Basename qw(basename);
 
 our @EXPORT = qw(switch_to_x11 wait_for_desktop ensure_unlocked_desktop);
 
 sub switch_to_x11 {
-    my @hdd = split(/-/, get_required_var('HDD_1'));
+    my @hdd = split(/-/, basename get_required_var('HDD_1'));
```

Comment:
better if something like this is split into its own commit

Reply:
Ok, I put it in its own branch

Reply:
I am sorry if I did not make myself clear but I meant *just* **commit**, not its own PR or branch